README.md

```diff
@@ -64,38 +68,45 @@ var handle = client.execute({
 handle.stream(function(err, data) { console.log(data); });
 ```
 
-Object `options` is an optional map and may contains following fields:
+The `options` object is an optional map and can contain following fields:
+
+| Field                    | Type     | Description                                                               | Default                                                                                            |
+|--------------------------|----------|---------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------|
+| `signalflowEndpoint`     | string   | Sets the SignalFlow endpoint.                                             | `wss://stream.us0.signalfx.com`. If you aren't in `us0`, change the realm.                         |
+| `apiEndpoint`            | string   | Sets the API endpoint.                                                    | `https://api.us0.signalfx.com`. If you aren't in `us0`, change the realm.                          |
+| `webSocketErrorCallback` | function | Sets the callback function that is invoked when a WebSocket error occurs. | Throws the WebSocket error. You can change the behavior to invoke a callback function instead.     |
```

Comment:
I'd change `Throws the WebSocket error.` to `If not set, throws the WebSocket error instead.` (or something akin to this wording)